scripts/check_if_for_format.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+REPO_ROOT="$(dirname "$0")"/..
+
+(
+cd $REPO_ROOT
+FORMATERROR=$(git grep -nIE "\<if\(" -- '*.h' '*.cpp')
+FORMATERROR=$FORMATERROR"\n"$(git grep -nIE "\<for\(" -- '*.h' '*.cpp')
+FORMATERROR=$FORMATERROR"\n"$(git grep -nIE "\<if\>\s*\(.*\)\s*\{\s*$" -- '*.h' '*.cpp')
```

Comment:
What about combining these to `(if|for)`?

Reply:
yeap